modules/uncertainty/R/generate_joint_ensemble_design.R

```diff
@@ -4,19 +4,20 @@
 #' are shared across sites to ensure consistent parameter sampling.
 #'
 #' @param settings A PEcAn settings object containing ensemble configuration
-#' @param sobol for activating sobol
 #' @param ensemble_size Integer specifying the number of ensemble members
+#' @param ensemble_samples list of ensemble parameters across PFTs. The default is NULL.
+#' @param sobol for activating sobol
```

Comment:
Add the `ensemble_samples` argument to ensure we can also pass the parameters directly.